supportedBackends/google/pipelines/v2alpha1/src/test/resources/reference.conf

```diff
@@ -1,2 +1,2 @@
 demo.dos.localization.docker-image = "somerepo/dos-downloader:tagged"
-demo.dos.localization.command-template = "/path/to/some_executable before args ${dosPath} middle args ${containerPath} ends args"
+demo.dos.localization.command-template = "/path/to/some_executable before args ${marthaUrl} ${dosPath} middle args ${containerPath} ends args"
```

Comment:
I think the command template should have `--env MARTHA_URL=<url>`. So, it can be something like `--env MARTHA_URL=${marthaUrl} /path/to/some_executable before args ${dosPath} middle args ${containerPath} ends args`